### DISCUSSION TITLE
Is there a way to play the animation without seeing what's outside the canvas???

### DISCUSSION THREAD

**Discussion by cvanalphen** [keiwando/toonsquid#493] [General]
Hi,

Is there an option to play the animation without seeing what's outside the canvas? Like a full screen player view?

**Comment by keiwando** [comment]
Do you mean without seeing any of the toolbars and buttons? That's not possible at the moment.
If you just want to completely hide all of the parts of your layers that are outside of the final export region on the canvas, then you can add a [camera layer](https://toonsquid.com/handbook/layers/camera/) and increase its "Outside Darkening Amount" property to 100%.

**Reply by cvanalphen** [reply]
Thank you! I will try this solution
